utils/vmd_utils.py

```diff
@@ -0,0 +1,9 @@
+import re
+import unicodedata
+import simplejson as jsoncode
+
+def urlify(s):
+    s = re.sub(r"[^\w\s\-]", '', s)
+    s = re.sub(r"\s+", '-', s).lower()
+    return jsoncode.dumps(unicodedata.normalize('NFKD', s).encode('ASCII', 'ignore')).strip('"')
```

Comment:
C'est un massacre là :sweat: Est ce que tu peux tester avec `''.join(c for c in unicodedata.normalize('NFD', s) if unicodedata.category(c) != 'Mn')`, ça devrait éviter le problème de bytearray que tu as eu avec `encode()` ?